src/components/AuthPage/ForgotPassword/index.jsx

```diff
@@ -16,151 +16,161 @@ import { Alert } from "@mui/material";
 import PropTypes from "prop-types";
 
 const useStyles = makeStyles({
-	root: {
-		padding: "2rem",
-		background: "theme.palette.background.paper",
-		border: "none",
-		boxShadow: "none",
-	},
-	heading: {
-		fontWeight: 600,
-	},
+  root: {
+    padding: "2rem",
+    background: "theme.palette.background.paper",
+    border: "none",
+    boxShadow: "none"
+  },
+  heading: {
+    fontWeight: 600
+  }
 });
 
 const ForgotPassword = ({
-	rootBackground = "rgba(0,0,0,.01)",
-	confirmationText = "We have sent you an email containing the link to reset your password .Please check your inbox including spams",
-	fontweight = "800",
-	buttonColor = "blue",
+  rootBackground = "rgba(0,0,0,.01)",
+  confirmationText = "We have sent you an email containing the link to reset your password .Please check your inbox including spams",
```

Comment:
Nitpik: 
Can we modify your text as below:
"We have sent you an email containing the link to reset your password. Please check your inbox including spams"

Change: 
Add full stop after password and one gap after full stop